CHANGELOG.md

```diff
@@ -62,7 +62,7 @@
 
 * **stripe:**  send statement descriptor to stripe (#707) (641c4d6d)
 *  use connector error handler for 500 error messages. (#696) (9fe20932)
-*  populate failed status and add bank_redirect (#674) (b1685ba1)
+*  populate failed status and add bank_redirect (#674)
```

Comment:
Should I maybe set up a [`git-cliff`](https://github.com/orhun/git-cliff) config file for generating changelogs?

Reply:
That helps a lots @SanchithHegde.